src/main/java/bootcamp/services/twitter4j/TwitterResourceService.java

```diff
@@ -1,73 +1,35 @@
 package bootcamp.services.twitter4j;
 
 import bootcamp.model.Message;
-import bootcamp.configuration.TwitterKeys;
 import bootcamp.model.Tweet;
 import bootcamp.model.User;
+import dagger.Module;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import twitter4j.Status;
 import twitter4j.StatusUpdate;
 import twitter4j.Twitter;
-import twitter4j.TwitterFactory;
-import twitter4j.conf.ConfigurationBuilder;
 
+import javax.inject.Inject;
 import java.util.List;
 import java.util.Optional;
 import java.util.stream.Collectors;
 
+@Module
```

Comment:
not sure why you made this into a `module`. Being an injectable service should be sufficient here.